Bottom: ee2ec1736de73c9f4e3690770e1545adb4e0c0f4
Top:    dde615809015620d4059fb2e2e1ebf625decb818
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-29 20:14:51 +0100

Refresh of hid-gtk-better-encapsulate-som

---

diff --git a/src/hid/gtk/ghid-layer-selector.c b/src/hid/gtk/ghid-layer-selector.c
index b575816..746b9d7 100644
--- a/src/hid/gtk/ghid-layer-selector.c
+++ b/src/hid/gtk/ghid-layer-selector.c
@@ -298,7 +298,6 @@ ghid_layer_selector_init (GHidLayerSelector *ls)
   GtkCellRenderer *renderer2;
   GtkTreeViewColumn *opacity_col;
   GtkTreeViewColumn *name_col;
-  GHidLayerSelector *ls;
 
   renderer1 = ghid_cell_renderer_visibility_new ();
   renderer2 = gtk_cell_renderer_text_new ();
